### PR TITLE
[DS-343]: [Documentation] Update Contribution Page on prima.design

### PR DESCRIPTION
**Pull request by pdjota**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/DS-343

This updates contribution guides after team discussion on scope of contribution

See also the related info in the getting started guide:
https://prima.zeroheight.com/styleguide/s/79372/p/0885af--collaborating-and-contributing/b/22aae3

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,35 +1,29 @@
-## 💜 Contributing
+# 💜 Collaborating and Contributing
 
-Pyxis is a living design system, undergoing continuous development, and you may find that it doesn't yet satisfy your requirements, be it a missing UI component, or a missing icon, or perhaps a typography setting you'd like to see changed...
+*Pyxis* is a living design system. As such it is in the process of continuous improvement,
```
Comment (brianvanburken) [inline CONTRIBUTING.md:3]:
```suggestion
*Pyxis* is a living design system. As such, it is in the process of continuous improvement, 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,35 +1,29 @@
-## 💜 Contributing
+# 💜 Collaborating and Contributing
 
-Pyxis is a living design system, undergoing continuous development, and you may find that it doesn't yet satisfy your requirements, be it a missing UI component, or a missing icon, or perhaps a typography setting you'd like to see changed...
+*Pyxis* is a living design system. As such it is in the process of continuous improvement, 
+it may not meet all your requirements: there could be an icon, a UI component or perhaps a typography setting you'd like to change.
+Maybe there's a problem which doesn't have a clear design solution or challenges with the implementation.
+In all cases team Pyxis is here to help.
```
Comment (brianvanburken) [inline CONTRIBUTING.md:6]:
```suggestion
In all cases, team Pyxis is here to help.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,35 +1,29 @@
...
 
-Following is a list of possible issues you'll encounter when integrating Pyxis in your application. Each issue has a related process defined by the Pyxis team in order to unblock you as soon as possible, letting you continue with your development.
+## Discuss with Team Pyxis your mockups or design challenges
 
-#### 1) I want to use Pyxis but I'm not following a mockup
+Pyxis implement solutions to many design problems.
+Mockups _can be validated with designers_ The _foundations, components and design tokens_ are the basic bricks for building everything.
```
Comment (brianvanburken) [inline CONTRIBUTING.md:11]:
```suggestion
Mockups _can be validated with designers_ The _foundations, components, and design tokens_ are the basic bricks for building everything.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,35 +1,29 @@
...
+Please check our [design site](https://prima.design/3794e337c/p/87a3b6-pyxis) where our documentation lives.
```
Comment (brianvanburken) [inline CONTRIBUTING.md:12]:
```suggestion
Please check our [design site](https://prima.design/3794e337c/p/87a3b6-pyxis), where our documentation lives.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,35 +1,29 @@
...
+Drop by in our slack channel [#team-pyxis](https://prima.slack.com/archives/C02GWSGR14Y) we are always keen to share time with you understanding you point of view, doing pairing.
```
Comment (brianvanburken) [inline CONTRIBUTING.md:13]:
```suggestion
Drop by our slack channel [#team-pyxis](https://prima.slack.com/archives/C02GWSGR14Y). We always try to share time with you, understand your perspectives, and pair.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,35 +1,29 @@
...
 
-That's a really bad situation! Pyxis is a Design System which obviously strictly relies on the guidelines defined by our Design team.
-Not only the mockup should _have been validated by designers_ but also needs to be created _by only using foundations, components and design tokens_ which are the basic bricks for building everything with Pyxis.
+We are happy to discuss the problem/solution and see how it fits it our design system guidelines and if it benefits other Pyxis users we are keen to include it.
```
Comment (brianvanburken) [inline CONTRIBUTING.md:15]:
```suggestion
We are happy to discuss the problem/solution and see how it fits our design system guidelines, and if it benefits other Pyxis users, we are keen to include it.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,35 +1,29 @@
...
 
-This is not a team's whim but a requirement which enforces UI/UX consistency, reduces code fragmentation and prevents unexpected behaviours when upgrading to future versions of Pyxis.
+## I've found a bug
 
-#### 2) I've got a mockup but it contains something that's missing from the current version of Pyxis
+This can happen. When you find a bug in Pyxis please [open an issue in Github](https://github.com/primait/pyxis/issues/new?assignees=&labels=bug%2Ctriage&template=bug_report.yml&title=%5BBug%5D%3A+) detailing:
```
Comment (brianvanburken) [inline CONTRIBUTING.md:19]:
```suggestion
Bugs can happen. When you find a bug in Pyxis, please [open an issue in Github](https://github.com/primait/pyxis/issues/new?assignees=&labels=bug%2Ctriage&template=bug_report.yml&title=%5BBug%5D%3A+) detailing:
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,35 +1,29 @@
...
 
-If you're sure that our documentation doesn't help you addressing this issue and you have a mockup which follows the guidelines defined in the previous case, you can _open an issue_, specifying the following information:
+- Version
+- Module (e.g. elm or react)
+- Expected behaviour
+- Clear steps to reproduce
+- Device(s) affected: which _browsers, resolutions or devices_ suffer from the bug
```
Comment (brianvanburken) [inline CONTRIBUTING.md:25]:
```suggestion
- Device(s) affected: which _browsers, resolutions, or devices_ suffer from the bug
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,35 +1,29 @@
...
+- Include screenshots or videos if you believe they can help us to understand and address the bug.
 
-1. a **short title** which defines the issue
-2. a **description** of what you need to do and how current Pyxis APIs prevent you from doing that
-3. **at least two reviewers** from the Pyxis team which can help you in the subsequent discussion on how this issue should be solved
-4. _[optional]_ **a suggestion** of how the issue's solution should be addressed. This will start the conversation. Keep in mind that this kind of solution should follow our guidelines and be something that benefits other Pyxis users. _Don't propose solutions that only satisfy your specific needs._
+Once you're doing feel free to ping us in our slack channel. We review the bugs periodically.
```
Comment (brianvanburken) [inline CONTRIBUTING.md:28]:
```suggestion
When done, feel free to ping us on our slack channel. We review the bugs periodically.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 10]
```diff
@@ -65,20 +65,8 @@ body:
     attributes:
       label: Screenshots
       description: |
-        If applicable, add screenshots to help explain your problem.
+        If applicable, add screenshots or video to help explain your problem.
```
Comment (brianvanburken) [inline .github/ISSUE_TEMPLATE/bug_report.yml:68]:
```suggestion
        If applicable, add screenshots or a video to help explain your problem.
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (pdjota) [reply CONTRIBUTING.md:11]: that comma is not needed

File: CONTRIBUTING.md (hunk 6 above)
Comment (pdjota) [reply CONTRIBUTING.md:15]: it is not a secondary phrase

**Review by brianvanburken — DISMISSED** [review]

**Review by emanuelemacri — CHANGES_REQUESTED** [review]
A minimal fix (only one period missing)

I do not agree much with the total elimination of the template for 'feature requests. Maybe we can simplify it, but it could be helpful in case of _, for example,_ an API is missing within a component or any DX enhancement is suggested.

What do you say, would you like to bring all these discussions on slack into #team-pyxis?

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 11]
```diff
@@ -1,63 +0,0 @@
...
-    id: feature_suggested_solution
-    attributes:
-      label: Suggested solution
-      description: Describe the solution you'd like.
-      placeholder: A clear and concise description of what you want to happen.
-    validations:
-      required: true
-  - type: textarea
-    id: feature_alternative_solution
-    attributes:
```
Comment (alessiosantangelo) [inline .github/ISSUE_TEMPLATE/feature_request.yml:37]: When we'll merge this PR we should also remember to remove the `request a feature` bullet point from [our Notion page](https://www.notion.so/Team-Pyxis-c36a2304069246cd89f8499030340a6e) (yellow box)

Comment (alessiosantangelo) [reply .github/ISSUE_TEMPLATE/feature_request.yml:37]: @pdjota how do you think we should deal with requests about missing api(s)? 
We should ask devs to write only on our public channel or something similar?

Comment (pdjota) [reply .github/ISSUE_TEMPLATE/feature_request.yml:37]: we will reintroduce it with `enhancement_request`

Comment (pdjota) [reply .github/ISSUE_TEMPLATE/feature_request.yml:37]: discussed with @emanuelemacri we decided to use the term enhancement and make the form simpler

**Review by emanuelemacri — APPROVED** [review]

**Pull request merged**